src/components/DataInput/changeHandlerFactories.ts

```diff
@@ -0,0 +1,26 @@
+export const handleChangeFactoryDropdownList = (onChange: any): any => {
+    return handleChangeFactory(onChange);
+};
+
+export const handleChangeFactoryFreeFormText = (onChange: any): any => {
+    return handleChangeFactory(onChange);
+};
+
+export const handleChangeFactoryRadioGroup = (onChange: any): any => {
+    return handleChangeFactory(onChange);
+};
+
+export const handleChangeFactoryCheckBox = (object: any, setObject: any): any => {
+    return (event: any) => {
+        setObject({
+            ...object,
+            [event.target.name]: event.target.checked,
+        });
+    };
+};
+
+export const handleChangeFactory = (onChange: any): any => {
+    return (event: any) => {
+        onChange((event.target as HTMLInputElement).value);
+    };
+};
```

Comment:
this entire file seems redundant, you can use the native event types instead

Reply:
Yeah, we were just trying to make things easier for you guys. Turns out you're too advanced for our niceties:(

I shall remove the file for you